contributors/devel/api_changes.md

```diff
@@ -852,7 +852,7 @@ The preferred approach adds an alpha field to the existing object, and ensures i
     * ensure the field is [optional](api-conventions.md#optional-vs-required)
         * add the `omitempty` struct tag
         * add the `// +optional` comment tag
-        * ensure the field is entirely absent from API responses when empty (if it is a struct type, it must be a pointer)
+        * ensure the field is entirely absent from API responses when empty (optional fields should be pointers, anyway)
```

Comment:
Is this true even for primitive types where omitempty/zero-values work as desired?

Reply:
yes.  Conventions doc says so, and the consistency is valuable.